### ISSUE TITLE
Accessibility: MAS40B: Inspect: 'Name' property is Null for Tree Item under 'BreakPoints' Section in left pane.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/vscode#17171]
VSCode Version:1.6.1
OS Version:1607

**Steps to Reproduce:**
1.Open Visual studio code editor.
2.Using "Ctrl+Shift+D" key navigate to the debug side view bar option.
3.Navigate to the Tree Item  under 'Break Points Section'.
4.Using Inspect, check its "Name" property.

**Expected Result:**
Tree Item under 'Break Points' Section should have a meaningful and appropriate "Name" property.
Eg: Name property should be "All Exceptions" for All exceptions Tree Item .

**Actual Result:**
"Name" property is null for Tree Item under 'Break Points' Section
Narrator Info:- Narrator announces as "1 of 3, Header, header updated, All Exceptions".
NVDA Info:- NVDA focus goes to the checkbox but announces nothing.
Observation: If the Breakpoint is Selected, then the Name Property is not Null.

**User Impact:**
Users relying on screen readers will get confused if the name property is not meaningful and inappropriate.

![breakpoint checkbox null name](https://cloud.githubusercontent.com/assets/22956130/21183484/257b1fd0-c22e-11e6-9f21-e35bf99d771b.png)

**Comment by isidorn** [comment]
@WiproPSAccTest do you also see this behavior for all the trees in vscode - namely in explorer or other trees in the debug view?
Is the name nicely set when you actually focus the brekapoint using the up / down key.

My assumpiton is that nvda computes the name from the aria-label property which we nicely set

**Comment by isidorn** [comment]
This should work nicely in latest - please try with vscode 1.8.1

**Comment by WiproPSAccTest** [comment]
@isidorn I checked in both the vs code editor Insider and vs code editor 1.8.1 edition. Both the edition I could able to reproduce the issue.

**Comment by isidorn** [comment]
@WiproPSAccTest then let me reopen and investigate again

**Comment by isidorn** [comment]
Still can not repro. You can see in the attached gif how I am using the keyboard up / down to navigate between the breakpoints and how each breakpoint is nicely read out.
Are you doing this with NVDA?
Pleae note that we **do not** support narrator atm.

![labels](https://cloud.githubusercontent.com/assets/1926584/22072321/5293a002-dda2-11e6-83ed-02c0d24025ec.gif)

**Comment by isidorn** [comment]
Closing until further details are provided
